Makefile

```diff
@@ -17,10 +17,10 @@ clean:
 distclean: clean
 	./rebar delete-deps
 
-testclean: clean
```

Comment:
How about leaving in this target should someone want to clean out the tests and add a lighter weight clean step as a dependency to test? Also then you could still test without cleaning if you so choose